src/Compilers/CSharp/Test/Semantic/Semantics/NullableConversionTests.cs

```diff
@@ -711,33 +711,15 @@ void Test()
     }
 }
 ";
-
-            var compilation = CreateCompilation(source);
-            compilation.VerifyDiagnostics(
-                // (15,13): error CS0030: Cannot convert type 'double' to 'int?'
-                //         i = (int?)double.MaxValue;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)double.MaxValue").WithArguments("double", "int?").WithLocation(15, 13),
-                // (16,13): error CS0030: Cannot convert type 'double' to 'int?'
-                //         i = (int?)double.NaN;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)double.NaN").WithArguments("double", "int?").WithLocation(16, 13),
-                // (17,13): error CS0030: Cannot convert type 'double' to 'int?'
-                //         i = (int?)double.NegativeInfinity;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)double.NegativeInfinity").WithArguments("double", "int?").WithLocation(17, 13),
-                // (18,13): error CS0030: Cannot convert type 'double' to 'int?'
-                //         i = (int?)double.PositiveInfinity;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)double.PositiveInfinity").WithArguments("double", "int?").WithLocation(18, 13),
-                // (22,13): error CS0030: Cannot convert type 'float' to 'int?'
-                //         i = (int?)float.MaxValue;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)float.MaxValue").WithArguments("float", "int?").WithLocation(22, 13),
-                // (23,13): error CS0030: Cannot convert type 'float' to 'int?'
-                //         i = (int?)float.NaN;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)float.NaN").WithArguments("float", "int?").WithLocation(23, 13),
-                // (24,13): error CS0030: Cannot convert type 'float' to 'int?'
-                //         i = (int?)float.NegativeInfinity;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)float.NegativeInfinity").WithArguments("float", "int?").WithLocation(24, 13),
-                // (25,13): error CS0030: Cannot convert type 'float' to 'int?'
-                //         i = (int?)float.PositiveInfinity;
-                Diagnostic(ErrorCode.ERR_NoExplicitConv, "(int?)float.PositiveInfinity").WithArguments("float", "int?").WithLocation(25, 13));
+            var compilation = DiagnosticsUtils.VerifyErrorsAndGetCompilationWithMscorlib(source,
```

Comment:
>DiagnosticsUtils [](start = 30, length = 16)

Can you please restore the previous style of diagnostic validation?  It is much easier to see if it looks right or not.